recipes/fiji/20141125/meta.yaml

```diff
@@ -0,0 +1,24 @@
+about:
+    home: http://fiji.sc/
+    license: GNU General Public License
+    summary: Fiji is an image processing package—a "batteries-included" distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.
+package:
+    name: fiji
+    version: 20141125
+build:
+  number: 1
```

Comment:
Let's see what the tests saying. But for the first version I'm fine with Linux support. It's sad this is not working under OS-X, but I can not help here unfortunately. :(